bin/ns-status.js

```diff
@@ -65,30 +60,29 @@ if (!module.parent) {
         .strict(true)
         .help('help');
 
-    var params = argv.argv;
-
-
-    var clock_input = params._.slice(0, 1).pop();
-    var iob_input = params._.slice(1, 2).pop();
-    var suggested_input = params._.slice(2, 3).pop();
-    var enacted_input = params._.slice(3, 4).pop();
-    var battery_input = params._.slice(4, 5).pop();
-    var reservoir_input = params._.slice(5, 6).pop();
-    var status_input = params._.slice(6, 7).pop();
-    var mmtune_input = params._.slice(7, 8).pop();
-    var uploader_input = params.uploader;
-
-    if (params._.length > 8) {
-        uploader_input = params.uploader ? params._.slice(7, 8).pop() : false;
-        mmtune_input = params._.slice(8, 9).pop();
+    var params = argv.argv,
+        inputs = params._,
+        clock_input = inputs[0],
+        iob_input = inputs[1],
+        suggested_input = inputs[2],
+        enacted_input = inputs[3],
+        battery_input = inputs[4],
+        reservoir_input = inputs[5],
+        status_input = inputs[6],
+        mmtune_input = inputs[7],
+        uploader_input = params.uploader;
```

Comment:
What's the advantage of making things into a comma-separated list vs. having a separate var for each line?

Reply:
Virtually none performance-wise, but I think it's more visually appealing if you're assigning a bunch of vars

Reply:
Ok, I'd rather optimize for minimizing errors and clarity of diffs than visual appeal.  I don't care if we standardize on lots of `var =` statements or putting commas on following lines, but have had enough typos due to copying and pasting lines without adjusting commas that I'd rather not go with a comma-separated list with the commas and semicolon at the end of the line.

Reply:
Fixed. Regarding errors, a potential risk with var assignments I've seen in a number of places is declaring vars mid block instead of at the top. This can lead to errors if devs don't know how hoisting works

https://www.w3schools.com/js/js_hoisting.asp